pages/advanced/api-new-networks.mdx

```diff
@@ -0,0 +1,58 @@
+import { Steps } from 'nextra/components'
+
+# New Supported Networks
+
+Safe's vision is to make every web3 account a smart account. Therefore, we prioritize teams and chains building with smart accounts, especially if they push account abstraction (for example, ERC-4337/EIP-1271 adoption) with Safe (learn more [here](https://docs.safe.global/home/4337-safe)).
+
+To have the full Safe stack running on your chain network, you need to consider the following steps:
+
+## Steps
```

Comment:
Thoughts on getting rid of this heading and updating the next ones to H2 and H3 respectively?

Reply:
Without the Stepper? I'm not sure I understand the suggestion

Reply:
Gotcha, I did not know about that. Could then rename the Steps heading to something descriptive like 'Add New Network' or something like that shows up with Google search?

Reply:
"Add new network" would be repetitive with the new title of the page